packages/mui-varnish/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@allenai/mui-varnish",
+  "version": "5.11.0",
+  "private": false,
+  "author": "Reviz",
+  "description": "Varnish theme and components that implement Google's Material Design.",
+  "main": "./src/index.ts",
+  "keywords": [
+    "react",
+    "react-component",
+    "mui",
+    "material-ui",
+    "material design",
+    "varnish"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/allenai/mui-varnish.git",
+    "directory": "packages/mui-varnish"
+  },
+  "license": "MIT",
```

Comment:
Our standard is Apache 2. Does this repository use MIT? We should stick with what MUI uses, but just want to make sure we're not deciding to use something else.